src/libasr/codegen/wasm_to_x64.cpp

```diff
@@ -109,7 +109,9 @@ class X64Visitor : public WASMDecoder<X64Visitor>,
     void gen_x64_bytes() {
         {   // Initialize/Modify values of entities for code simplicity later
 
-            m_a.update_asm("BITS 64\n\n"); // Update initial value of asm text as per X64 text format
+            m_a.update_asm(""); // delete previous asm code
+            m_a.append_asm("BITS 64\n");
+            m_a.append_asm("\torg " + i2s((uint64_t)m_a.origin()) + "\n\n"); // specify origin info
```

Comment:
Instead, let's change this:
```
    X86Assembler(Allocator &al) : m_al{al} {
```
To:
```
    X86Assembler(Allocator &al, bool bits64) : m_al{al} {
```
where `bits64` is true for 64 bit and false for 32 bit.

Reply:
Got it. I added the `64bits` parameter to `X86Assembler()` constructor.